__test__/utils/index.test.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidId } from "../../src/lib/utils";
+
+describe("isValidId", () => {
```

Comment:
Could you add an extra test case in which the length is right but we have invalid characters (e.g. `-`)?
That would give a higher confidence on the correctness of this function. 😄

Reply:
fixed